workflow.yml

```diff
@@ -0,0 +1,57 @@
+name: Stale Issue Workflow
+
+on:
+  # Allows manually running
+  # https://docs.github.com/en/actions/reference/events-that-trigger-workflows#manual-events
+  workflow_dispatch:
+  
+  schedule:
+    # Runs at 08:00 UTC every day
+    # https://docs.github.com/en/actions/reference/events-that-trigger-workflows#schedule
+    - cron: '0 8 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      # https://github.com/actions/stale
+      - uses: actions/stale@v3
+        with:
+          repo-token: ${{ github.token }}
+          # do not manage PRs
+          days-before-pr-stale: -1
+          days-before-pr-close: -1
+          # stale issue config
+          days-before-issue-stale: 0 # 90
+          days-before-issue-close: 0 # 21
+          stale-issue-message: |
+            Hello, I'm a bot and I thank you in the name of the community for opening this issue.
+
+            To help our human contributors focus on the most relevant reports, I check up on old issues to see if they're still relevant.
+            This issue has had no activity for 90 days, and so I marked it as stale.
+
+            The community would appreciate your effort in checking if the issue is still valid. If it isn't, please close it.
```

Comment:
The community would appreciate if you could check if the issue still persists. If it isn't, please close it.